src/js/types.ts

```diff
@@ -1,99 +1,100 @@
 export interface CindyType {
-  ctype: string;
-  value?: Record<string, any>;
+    ctype: string;
+    value?: Record<string, any>;
 }
 
 export interface CindyNumber extends CindyType {
-  usage?: "Angle";
-  ctype: "number";
-  value: {
-    real: number;
-    imag: number;
-  };
+    usage?: "Angle";
+    ctype: "number";
+    value: {
+        real: number;
+        imag: number;
+    };
 }
 
-export interface Nada {
```

Comment:
This is becoming very verbose... I would prefer to keep such names short. Can we keep using Nada?

Reply:
Sure. Are there any preferences for the interface names?

e.g.
CindyNumber, CNumber, CindyNum?
CindyList, CList, CindyLst?

Reply:
Prepend with `C` or `CS`. Probably `CS` is better.  

I had a look at the Cinderella code, there we use (thanks to packages) normal names for math:

```
math (beta) % ls
Assignable.java			Int.java			PolarConversion.java
AssignableAdapterB.java		Mat.java			RMat.java
AssignableAdapterI.java		Mat2.java			Tracable.java
AssignableVector.java		MathHelper.java			TransformationHelper.java
Bool.java			Matrix3D.java			TypedObject.java
Complex.java			MovableReference.java		Vec.java
Doub.java			MyMath.java			Vec2.java
FormulaValue.java		Nada.java
GetSetAdapter.java		Point2D.java
```

For geometric elements we usually prepend with `PG`. And CindyScript operators are called `OpSomething`. And Lists are called CEnum.